jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -1164,7 +1173,23 @@ public Method getDeclaredMethod(String name, Class<?>... parameterTypes) throws
  */
 @CallerSensitive
 public Class<?> getDeclaringClass() {
-	Class<?> declaringClass = getDeclaringClassImpl();
+	if (cachedDeclaringClassOffset == -1) {
+		long localCachedDeclaringClassOffset;
+		try {
+			Field cachedDeclaringClassField = Class.class.getDeclaredField("cachedDeclaringClass"); //$NON-NLS-1$
+			localCachedDeclaringClassOffset = getUnsafe().objectFieldOffset(cachedDeclaringClassField);
+		} catch (NoSuchFieldException e) {
+			throw newInternalError(e);
+		}
+		/*[IF Sidecar19-SE]
```

Comment:
Pls add `*/` at end of this JPP decoration (and following similar cases) such that following `Java 9` implementation can be checked by editor compiler.